metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,80 @@
+# Title
+
+Utah Roads
+
+## ID
+
+a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
+
+## Brief Summary
+
+The latest geospatial information for all roads and highways in Utah.
```

Comment:
**thought**: it might be helpful to know how this is intended to be used. 

On the category index pages, https://gis.utah.gov/products/sgid/transportation/, these cards show the brief summary. 

<img width="988" alt="image" src="https://github.com/agrc/metadata-asset-tracking-tool/assets/325813/4b25dc8f-624a-41c0-868e-101594e08076">

Then if you click the card, the summary is shown.

<img width="995" alt="image" src="https://github.com/agrc/metadata-asset-tracking-tool/assets/325813/7a6373f4-e572-49bb-a6e8-893484115188">

In my opinion, it's fine if they overlap each other. 

For instance, I would rather read this on the index card

> A multi-purpose statewide roads and highways dataset for cartography, routing, and range-based address location.

Reply:
I think that's a great idea. Being able to have them overlap opens up the possibilities quite a bit.